Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -2692,7 +2712,180 @@ def main():
             raise
 
 
-from ServiceNowApiModule import *  # noqa: E402
+### GENERATED CODE ###: from ServiceNowApiModule import *  # noqa: E402
```

Comment:
Below OAuth class `ServiceNowClient` is already defined in ServiceNowApiModule automation script and can be imported using `from ServiceNowApiModule import *`.